java/core/src/java/org/apache/orc/impl/WriterImpl.java

```diff
@@ -645,6 +645,8 @@ public void appendUserMetadata(List<OrcProto.UserMetadataItem> userMetadata) {
     return ReaderImpl.deserializeStats(builder.getStatisticsList());
   }
 
+  // TODO: remove this
```

Comment:
Remove the changes in this file. We do not want a guice dependency.

Reply:
Removed the attribute, clarified the TODO